public/app/features/provisioning/types.ts

```diff
@@ -11,4 +11,4 @@ export interface ProvisioningPreview {
   ref?: string;
 }
 
-export type WorkflowOption = 'branch' | 'push';
+export type WorkflowOption = 'branch' | 'write';
```

Comment:
Can we get this automatically generated? I tried with the `enum` annotation, but it didn't work.

Reply:
it exists here: https://github.com/grafana/grafana/pull/101031/files#diff-197a82c657626e5548eb4609f4ae902574bb297f2a78dc5b6f5603b66954f3b4R806

```
workflows?: ('branch' | 'write')[];
```
🤷🏻  but there must be some better way to get enums generated

Reply:
Ideally the generated type would be 
```ts
type Workflow = 'branch' | 'write';
workflows: Workflow[]
```